locales/en.default.schema.json

```diff
@@ -866,7 +866,7 @@
             },
             "show_gift_card_recipient": {
               "label": "Show recipient form for gift card products",
-              "info": "When enabled, gift card products can optionally be sent to a recipient with a personal message."
+              "info": "When enabled, gift card products can optionally be sent to a recipient with a personal message.[Learn more](https://help.shopify.com/en/manual/products/gift-card-products/add-update-gift-card-products)"
```

Comment:
This is for the featured product section but the setting isn't present on that section. So not sure if we should remove it or leave it there for when it's added in the future 🤔  cc: @antiBaconMachine